src/js/common/schemaform/sip-api.js

```diff
@@ -0,0 +1,31 @@
+import Raven from 'raven-js';
```

Comment:
I wasn't sure where to put this, so I made a new file.

Reply:
I'm not sure. It's hard to use apiRequest in many places since we have custom logic for tracking failures. So we end up repeating that everywhere. It might make more sense to build an api file like this where we put fetch requests with no redux logic.

Reply:
Good point, this is a more elegant/extensible/lightweight option.